json/flagd_definitions.go

```diff
@@ -4,3 +4,6 @@ import _ "embed"
 
 //go:embed flagd-definitions.json
 var FlagdDefinitions string
+
+//go:embed targeting.json
+var Targeting string
```

Comment:
Now we export a const for the `Targeting` schema as well.

Reply:
Nice. We should follow up and add validations in flagd for flag parsing [^1]

[^1]: https://github.com/open-feature/flagd/blob/main/core/pkg/evaluator/json.go#L404-L406